package.json

```diff
@@ -31,6 +31,7 @@
     "firebase": "^10.7.2",
     "ini": "^4.1.3",
     "inquirer": "^9.2.15",
+    "jsdom": "^26.1.0",
```

Comment:
Is there a reason why you need `jsdom` to be added back here? I took it out in https://github.com/p0-security/p0cli/pull/186 in favor of `cheerio` because it was causing a build warning for the standalone CLI.

If you need some simple HTML parsing or basic DOM manipulation, `cheerio` should be sufficient. "jsdom" is a little overkill IMO.